__test__/parsing.spec.ts

```diff
@@ -169,3 +169,16 @@ test('Should throw error on invalid individual recurrence rule week start', () =
     'RRule parsing error: `Invalid` is not a valid weekday start. Valid values are `MO`, `TU`, `WE`, `TH`, `FR`, `SA` and `SU`.',
   );
 });
+
+test('Should be able to parse rule set without dtstart', () => {
+  const set = new RRuleSet(
+    new RRuleDateTime(new Date('1997-09-02T09:00:00-04:00'), 'US/Eastern'),
+  );
+  set.setFromString(
+    'RRULE:FREQ=WEEKLY;INTERVAL=2;UNTIL=19971224T000000Z;WKST=SU;BYDAY=MO,WE,FR',
+  );
+  const asString = set.toString();
+  expect(asString).toBe(
+    'DTSTART;TZID=US/Eastern:19970902T090000\nRRULE:FREQ=WEEKLY;UNTIL=19971224T000000Z;INTERVAL=2;WKST=Sun;BYHOUR=9;BYMINUTE=0;BYSECOND=0;BYDAY=MO,WE,FR',
```

Comment:
The invalid `Sun` will be fixed once this lands: https://github.com/fmeringdal/rust-rrule/pull/111

Reply:
Can we than add a test case without "Sun"?

Reply:
Yeah we could, OTOH this is an issue in many other tests already.

Reply:
Yeah, you are right, didn't notice that. Let's than keep it as it is